index.d.ts

```diff
@@ -289,6 +289,19 @@ export const PixiFiber: Reconciler<any, any, any, any>;
 // stage
 export class Stage extends React.Component<_ReactPixi.IStage> {}
 
+type SpineContainerStatic = {
+  Animation: any;
+}
+
+type SpineContainerInterface<T> = React.ComponentClass<any, any>
+
+export const SpineContainer: SpineContainerInterface<SpineContainerStatic> & {
+  Animation: any;
+  Slot: any;
+  Bone: any;
+}
+
```

Comment:
I think these types can be improved. Try to export SpineContainer on top-level and put everything else in the private `_ReactPixi` namespace. Also to assist users try to prevent `any` as much as possible.

example: _(pseudo code)_

```ts
declare namespace _ReactPixi {
  type ISpineContainer = IContainer & {
    mixes: Array<{
      from: string;
      to: string;
      time: number;
    }>;
    animations: string[];
    ....
  }
}

export class SpineContainer extends React.Component<_ReactPixi.ISpineContainer> {}
```